configs/defaults.yaml

```diff
@@ -20,6 +20,13 @@
 #   # Write logs to standard output in human-readable form
 #   stdout: false
 
+# AWS provisioning account (EC2, STS...)
+# aws:
+#   region: us-east-1
+#   key:
+#   secret:
+#   session:
+
 # AWS CloudWatch logger
 # cloudwatch:
 #   enabled: false
```

Comment:
The patch introduces second AWS credentials, the one we already have are only for cloudwatch logging. This one is used for actual provisioning (the master RH account).